documentation.html

```diff
@@ -312,19 +316,287 @@ <h4 class="component__sub__title">Floating Buttons </h4>
                 <div class="code-display">
                     <iframe
                         src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=one-light&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=0px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=143%25&si=false&es=2x&wm=false&code=%253Cbutton%2520class%253D%2522btn%2520btn-float%2522%253E%2520%253Ci%2520class%253D%2522fas%2520fa-plus%2522%253E%253C%252Fi%253E%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520btn-float%2522%253E%2520%253Ci%2520class%253D%2522fas%2520fa-arrow-up%2522%253E%253C%252Fi%253E%253C%252Fbutton%253E%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520"
-                        style="width: 1024px; height: 473px; border:0; transform: scale(1); overflow:hidden;"
+                        style="border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+            </div>
+
+            
+
+            <div class="line"></div>
+
+
+
+            <!-- Cards -->
+            <div id="Card" class="card__main">
+                <h1 class="component__name">Cards</h1>
+                <p class=" component__desc">The card component can contain a variety of content, including a heading, image, content and buttons.
+                    Card is basically Well organized collection of data related to specific topic which is easy to understand. 
+                </p>
+                <p class="component__sub__desc">we have different types of card so just copy the code whichever 
```

Comment:
If you are using BEM, you can't add two-element at a single classs. (component__sub__desc) Instead of this you can write (component__sub-desc)